README.rst

```diff
@@ -28,241 +28,273 @@ pywbemtools: Python tools for communicating with WBEM servers
-Management Initiative Specification(SMI-S).
+CIM/WBEM standards are used for a wide variety of systems management tasks
+in the industry including DMTF management standards and the `SNIA`_
+Storage Management Initiative Specification (`SMI-S`_).
 
-Pywbemcli provides the functionality to :
+Pywbemcli provides access to WBEM servers from the command line.
+It provides functionality to:
 
-* Explore the CIM data of WBEM Servers. It can manage/inspect the CIM model
-  components including CIM classes, CIM instances, and CIM qualifiers and execute
-  CIM methods and queries on the WBEM server. It implements subcommands to
-  execute the CIM-XML operations defined in the DMTF specification
-  CIM Operations Over HTTP(DSP0200_).
+* Explore the CIM data of WBEM servers. It can manage/inspect the CIM model
+  components including CIM classes, CIM instances, and CIM qualifiers and
+  execute CIM methods and queries on the WBEM server.
 
-* Inspect/manage WBEM server functionality such as CIM namespaces, WBEM
-  registered profiles, and other server information.
+* Execute specific CIM-XML operations on the WBEM server as defined in `DMTF`_
+  standard `DSP0200 (CIM Operations over HTTP)`_.
 
-* Capture detailed information on interactions with the WBEM server including
-  time statistics.
+* Inspect and manage WBEM server functionality including:
 
-.. _DSP0200: https://www.dmtf.org/sites/default/files/standards/documents/DSP0200_1.4.0.pdf
+  * CIM namespaces
+  * Advertised WBEM management profiles
+  * WBEM server brand and version information
 
-Installation Requirements
--------------------------
+* Capture detailed information on CIM-XML interactions with the WBEM server
+  including time statistics and details of data flow.
 
-1. Python 2.7, 3.4 and higher
+* Maintain a file with persisted WBEM connection definitions so that pywbemcli
+  can access multiple WBEM servers by name.
+
+* Provide both a command line mode and an interactive mode where multiple
+  pywbemcli commands can be executed within the context of a WBEM server.
+
+* Use an integrated mock WBEM server to try out commands. The mock server
+  can be loaded with CIM objects defined in MOF files or via Python scripts.
 
-2. Operating Systems: Linux, MacOS, Windows(native) and Windows(Cygwin)
 
 Installation
 ------------
 
-The quick way:
+Requirements:
 
-.. code-block:: bash
-
-    $ pip install pywbemtools
+1. Python 2.7, 3.4 and higher
 
-For more details, see the `Installation section`_ in the pywbemcools
-documentation.
+2. Operating Systems: Linux, OS-X, native Windows, UNIX-like environments on
+   Windows (e.g. Cygwin)
 
-.. _Installation section: https://pywbemtools.readthedocs.io/en/stable/introduction.html#installation
+The quick way for Linux and OS-X:
 
-Documentation
--------------
+.. code-block:: bash
 
-The latest pywbemtools documentation is available on ReadTheDocs:
+    $ wget -q https://raw.githubusercontent.com/pywbem/pywbem/master/pywbem_os_setup.sh
```

Comment:
You don't have to get it.  For now we use our own shell cmd and wget the .bat file from pywbem

Reply:
If a user installs from Pypi, they don't have the github repo checked out. So they need to get the script, just not from pywbem but from pywbemtools.
DONE - Changed to get from pywbemtools.

Reply:
I was really trying to get this down to just: 

   $ pip install pywbemtools

Forgot we were doing our installs from the git repo in our tests.

Since it turns out most users at this level are operating from Windows, we need to add the window alternative (at least the native windows but not about cygwin) to this also, which, of course is a mess because of wget.  I assume this means:

'''

install Chocolatey
$ choco install wget
$ wget -q https://raw.githubusercontent.com/pywbem/pywbemtools/master/pywbem_os_setup.bat
$ pywbem_os_setup.sh
pip install pywbemtools

As an alternative could we put the scripts into PyPi?????

Reply:
The Windows alternative is already described in the installation chapter of the docs. I have made an update to this PR that includes installation on Windows (but wget is stated as a prerequisite).

We cannot put the scripts into the Pypi package because I did not find a way to let them be executed upon install. We can of course just have them there as a script, but that breaks the pip install into pip download, execute the script, pip install, which is not less ugly, particularly since pip download only exists since pip version soandso.

Reply:
I agree with you comments on the scripts but I makes what should be simple for the user, in particular windows, messy.

Darn.  Then I only have one more comment.  It comes down to "hey windows user, you figure out how to install wget. Not our problem.". The choices are a) leave it to them, add the install of Chocolatey which again leads them off to clipping and pasting a script, 3) a sentence about where they can get wget (install with chocolatey, https://eternallybored.org/misc/wget/, or possibly other sites.

I will go with any of them but from what I have seen, the first users will want to use pywbemtools, not fight installing it.

Reply:
I can make the following changes to address your comments, but I would rather do them in a separate PR because they affect not just the READMEs but also the install section in the docs:
* On wget: I can add the one line to install wget using choco.
* On need for pywbem_os_setup scripts: For just installation, they are needed on all OS platforms but only with Python 2. I can update the install description accordingly.
  Note: For development, they are needed on all OS platforms with all Python versions, but that is covered in the how to develop section.